addons/zeus/config.cpp

```diff
@@ -19,6 +19,7 @@ class CfgPatches {
             QGVAR(moduleToggleNvg),
             QGVAR(moduleToggleFlashlight),
             QGVAR(moduleSimulation),
+            QGVAR(moduleCargoParadrop),
```

Comment:
Move down to `GVAR(cargo)` like `moduleLoadIntoCargo`. That handles requiredAddons.